apps/dashboard/src/lib/features/exercise-template/api/template.svelte.ts

```diff
@@ -0,0 +1,38 @@
+import { BaseApi, classroomio } from '$lib/utils/services/api';
+import type { TTemplate } from '@cio/db/types';
```

Comment:
Thinking about it now, we should also update the table name to `exercise_template` because I see us having several kinds of templates with different structures in the future.